src/main/resources/application.yml

```diff
@@ -6,7 +6,7 @@ camel:
     json-jackson:
       auto-discover-object-mapper: true
 
-dfspids: "DFSPID"
+dfspids: "gorilla"
```

Comment:
this change can be done from deployment, it is not necessary to be part of the code.